build-system/tasks/storybook.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const argv = require('minimist')(process.argv.slice(2));
+const {execOrDie} = require('../common/exec');
+
+let storybookArgs = '--quiet';
+if (argv.port) {
+  storybookArgs += ` -p ${argv.port}`;
+}
+
+/**
+ * Simple wrapper around the storybook start script
+ * for AMP components (HTML Environment)
+ */
+async function storybookAmp() {
+  execOrDie(
+    'node_modules/.bin/start-storybook -c ./tools/storybook/amp-env ' +
+      storybookArgs
+  );
+}
+
+/**
+ * Simple wrapper around the storybook start script.
+ */
+async function storybookBento() {
```

Comment:
Thoughts on `storybookPreact` instead with s/Bento/Preact in all other places (?)

Technically Bento mode is using the AMP custom element in a non-AMP document and Preact mode is using the Preact Component. If we ever want to have storybook for all three modes then we should name them accordingly from the start.

Reply:
Done